editor/connections_dialog.cpp

```diff
@@ -1140,6 +1140,7 @@ void ConnectionsDock::_open_connection_dialog(TreeItem &p_item) {
 	cd.signal = StringName(signal_name);
 	cd.target = dst_node;
 	cd.method = ConnectDialog::generate_method_callback_name(cd.source, signal_name, cd.target);
+	cd.flags = CONNECT_NO_EDITOR; // Always enable when connecting from the editor.
```

Comment:
I think the comment warrants an explanation as to _why_ rather than _what_. Why is this flag enabled in the editor? What does this achieve?